woocommerce/woocommerce-admin
# Update to Color Studio v2.1.0

Partially Fixes #2574.

This PR updates the color-studio package to v2.1.0. We were on 1.0.5 before so this contained breaking changes (all the variable names changed). The colors and schemes have also been iterated on as well.

I originally was going to do this just for Onboarding, but I went ahead and applied these to the general components, analytics and dashboard sections as well so we didn't have to keep different color schemes for different parts of the same plugin. So we might want a quick design peak at this (cc @LevinMedia ).

Please note that Calypso changes will be done in a separate PR.

### Screenshots

<img width="766" alt="Screen Shot 2019-08-26 at 1 45 27 PM" src="https://user-images.githubusercontent.com/689165/63711465-7a157c00-c809-11e9-9e5d-4af29941a994.png">

<img width="915" alt="Screen Shot 2019-08-26 at 1 44 41 PM" src="https://user-images.githubusercontent.com/689165/63711477-80a3f380-c809-11e9-8d20-a5d78012c919.png">

<img width="920" alt="Screen Shot 2019-08-23 at 11 47 59 AM" src="https://user-images.githubusercontent.com/689165/63605725-8943c200-c59c-11e9-8493-c53bad933252.png">

### Detailed test instructions:

* Close out of your current npm process, and run `npm install`.
* Make sure the application boots up and you do not receive any undefined variable errors when building the CSS.
* Click around various parts of the WooCommerce Admin experience. See the new colors and make sure everything looks OK.
* In the case of onboarding, you can compare to the Figma files.

### Changelog Note:

Updated to color-studio v2 and refreshed the color scheme
